benchmark/unwind_bench.cc

```diff
@@ -7,7 +7,11 @@
 #include <stdlib.h>
 #include <stdio.h>
 #include <stdint.h>
+#if defined(__QNXNTO__)
+#include <ucontext.h>
+#else
 #include <sys/ucontext.h>
```

Comment:
I think it is worth just dropping includes of ucontext stuff here. It is conditionally included below only for platforms that we're able to use it.